docs/AutocompleteInput.md

```diff
@@ -384,6 +390,14 @@ const optionRenderer = choice => `${choice.first_name} ${choice.last_name}`;
 `optionText` is also useful when the choices are records [fetched from another resource](#fetching-choices), and `<AutocompleteInput>` is a child of a [`<ReferenceInput>`](./ReferenceInput.md). 
 
 ```jsx
+// in ./index.js
+import {Admin, Resource } from 'react-admin';
+
+<Admin {...adminProps}>
+    <Resource name="authors" recordRepresentation={record => record.name} />
+</Admin>
```

Comment:
That's not valid React!

```suggestion
import { Admin, Resource, ListGuesser } from 'react-admin';
import { dataProvider } from './dataProvider';
import { PostCreate } from './PostCreate';

export const App = () => (
    <Admin dataProvider={dataProvider}>
        <Resource name="posts" list={ListGuesser} create={PostCreate} />
        <Resource name="authors" recordRepresentation={record => record.name} />
    </Admin>
    );
```

Also, I would put that snippet *after* the PostCreate snippet.